code/src/components/InputForm[Konflikt].js

```diff
@@ -0,0 +1,14 @@
+import React, { useEffect, useState } from 'react'
+
+const InputForm = () =>{
+    
+    return (
+    <form>
+        <h1>What ismax="" name="comment" form="usrform">
+    Enter text here...</textarea>
+    <button onclick="myFunction()"> A</button>
+    </form>
+    )
+}
+
+export default InputForm;
```

Comment:
I don't really understand what this component is for, and why there is no end tag for h1 and no opening tag for textarea?

Reply:
Yep, needs to be fixed, there are many mistakes here

Reply:
Seemed that there was a corrupt file so I removed it and now it works just as before 👍